.travis.yml

```diff
@@ -29,4 +29,4 @@ deploy:
   on:
     tags: true
     repo: earthlab/abc-classroom
-    python: 3.6
+    python: 3.7
```

Comment:
pushing to pypi on build 3.7 instead of 3.6 given 3.6 with go away at some point... if 3.9 is around the corner.